bin/src/main.rs

```diff
@@ -64,7 +79,12 @@ fn eth_test_simple() {
 
         println!("Running powdr-riscv executor in fast mode...");
         let start = Instant::now();
-        let (trace, mem) = riscv_executor::execute::<GoldilocksField>(&asm_contents, &data, &vec![], riscv_executor::ExecMode::Fast);
+        let (trace, mem) = riscv_executor::execute::<GoldilocksField>(
+            &asm_contents,
+            &data,
+            &default_input(),
```

Comment:
When the bootloader is included, we have to pass the default inputs rather than an empty array.